packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -42,13 +42,13 @@ rageTradeAccountId = "22262"
 zeuzAccountId = "46092"
 
 [var.cannonClonePackages]
-coreRouterPackage = "reya-core:1.0.15@router"
+coreRouterPackage = "reya-core:1.0.16@router"
```

Comment:
I would change the overall version = "1.0.48" too

Reply:
done https://github.com/Reya-Labs/reya-deployments/pull/262/commits/acc233178947f51f490c4cf7165b1b3f182d3aee